profile/latest/advisor/management/advisor/models.go

```diff
@@ -0,0 +1,40 @@
+package advisor
+
+import (
+	 original "github.com/Azure/azure-sdk-for-go/service/advisor/management/2017-04-19/advisor"
```

Comment:
Can't do it! With multi-APIVersion support, they must be separate by design. :) Composite swaggers were a huge mistake that should never have come around.